src/authenticated/project/ProjectsListPage.tsx

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import LoadingIndicator from '../../shared-components/LoadingIndicator';
 import ProjectCard from './ProjectCard';
+import { useCustomQuery } from '../../utils/hooks';
 import {
-  ProjectCardsHOC,
-  ProjectCardsAllProjects
+  ProjectCardsAllProjects,
+  ProjectCardsQuery,
+  ProjectCardsDocument
 } from '../../generated-models';
 
 interface ProjectsListProps {
   projects: ProjectCardsAllProjects[];
   loading: boolean;
 }
 
-const ProjectsList: React.SFC<ProjectsListProps> = ({ projects, loading }) => {
-  if (loading) {
-    return <LoadingIndicator />;
-  }
+const ProjectsListPage: React.FC<ProjectsListProps> = () => {
+  const { data } = useCustomQuery<ProjectCardsQuery>(ProjectCardsDocument);
+  if (!data || !data.allProjects) return <LoadingIndicator />;
```

Comment:
This doesn't follow the same pattern as the other cases of `useCustomQuery`. In the others you return `null`, but here a loading indicator is returned. As a user, this is more confusing than being given an empty screen because the loading indicator implies that the data is still coming.

Reply:
Good catch.